recipients_app/ios/Podfile

```diff
@@ -30,7 +30,7 @@ flutter_ios_podfile_setup
 target 'Runner' do
   use_frameworks!
   use_modular_headers!
-  pod 'FirebaseFirestore', :git => 'https://github.com/invertase/firestore-ios-sdk-frameworks.git', :tag => '10.24.0'
+  pod 'FirebaseFirestore', :git => 'https://github.com/invertase/firestore-ios-sdk-frameworks.git', :tag => '10.25.0'
 
```

Comment:
I had to use Firebase iOS SDK 10.25.0. I do not see any problems with this version update.
See this page for the changed log of 10.25.0: https://firebase.google.com/support/release-notes/ios#version_10250_-_april_30_2024